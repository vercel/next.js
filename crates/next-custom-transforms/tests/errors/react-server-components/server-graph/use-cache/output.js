export const runtime = 'edge';
// The following route segment configs are currently allowed:
export const dynamic = 'force-dynamic';
export const dynamicParams = false;
export const fetchCache = 'force-no-store';
export const revalidate = 1;
