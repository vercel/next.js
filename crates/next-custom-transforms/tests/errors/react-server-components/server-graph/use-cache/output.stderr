  x Route segment config "runtime" is not compatible with `nextConfig.experimental.useCache`. Please remove it.
   ,-[input.js:1:1]
 1 | export const runtime = 'edge'
   :              ^^^^^^^
 2 | // The following route segment configs are currently allowed:
   `----
